main.go

```diff
@@ -1,5 +1,16 @@
 package main
 
-func main() {
+import (
+	"log"
+
+	"github.com/SaiKumarMalve/Gator-News/bd"
+	"github.com/SaiKumarMalve/Gator-News/handlers"
+)
 
+func main() {
+	if bd.CheckConnection() == 0 {
```

Comment:
wait for my changes to be merged to main

Reply:
I merged my changes to main.
Everything looks good. Go Ahead and merge your changes.

Reply:
Okay. Sure

Reply:
I merged my changes to main.
Everything looks good. Go Ahead and merge your changes.